### PR TITLE
Add pair result types to export

### PR DESCRIPTION
**Pull request by DHHZ**
Small patch to add pair types to the function signature exporter. 

All the other pieces to translate tuple return types to Python were already in place.

### CODE REVIEW

File: src/lib/Export.hs [hunk 1]
```diff
@@ -143,6 +143,10 @@ prepareFunctionForExport env nameStr func = do
           elemTy' <- substBuilder (b@>Var i) elemTy
           createDest (idx <> Nest (Bind i) Empty) elemTy'
         return (Con $ TabRef destTab, exportResult)
+      PairTy a b -> do
```
Comment (apaszke) [inline src/lib/Export.hs:146]: Can you add a check that makes sure that pair types only appear on the outermost level of recursion? I think it's enough to add `| idx == Empty` after `PairTy a b` (to add an extra guard that we're not recursing through any table arrows).

**Review by apaszke — APPROVED** [review]
Wow that looks great. LGTM but please add a check

File: src/lib/Export.hs (hunk 1 above)
Comment (DHHZ) [reply src/lib/Export.hs]: Yep, done.

**Comment by apaszke** [comment]
Perfect, thank you!

**Pull request merged**